test/unit-tests/diagnostics.test.ts

```diff
@@ -243,6 +243,11 @@ suite('Diagnostics', async () => {
     expect(path.posix.normalize(diag.file)).to.eq(diag.file);
     expect(path.posix.isAbsolute(diag.file)).to.be.true;
   });
+  test('Parsing non-diagnostic', () => {
+    const lines = ['/usr/include/c++/10/bits/stl_vector.h:98:47: optimized: basic block part vectorized using 32 byte vectors'];
+    feedLines(build_consumer, [], lines);
+    expect(build_consumer.compilers.gcc.diagnostics).to.have.length(0);
+  });
   test('Parsing linker error', () => {
```

Comment:
I think you need .to.have.length(1);

Reply:
I figured out what I want. I needed to `resolveDiagnostics`.